app/components/owned-card-list-by-discipline.cjsx

```diff
@@ -0,0 +1,89 @@
+counterpart = require 'counterpart'
```

Comment:
This file can be removed - please do so by merging https://github.com/zooniverse/Panoptes-Front-End/pull/2369